traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.anyMap.tpl.html

```diff
@@ -59,429 +61,387 @@
+                                <aside>
+                                    <h6>Note</h6>
+                                    <p>This only works with Edge-tier cache servers running <abbr title="Apache Traffic Server">ATS</abbr> version 6 and earlier. This <em>must</em> be empty if using <abbr title="Apache Traffic Server">ATS</abbr> 7 and / or the <a href="https://docs.trafficserver.apache.org/en/7.1.x/admin-guide/plugins/cachekey.en.html" target="_blank">cachekey plugin.</a></p>
+                                    <br>
+                                    <br>
+                                    See <a href="https://docs.trafficserver.apache.org/en/6.2.x/admin-guide/plugins/cacheurl.en.html" target="_blank">ATS documentation on cacheurl</a>
+                                </aside>
+                                <aside class="warning">
+                                    <h6>Warning</h6>
+                                    <p>This field is deprecated, and is subject to removal in upcoming releases. It is recommended that this be left blank.</p>
+                                </aside>
+                            </div>
+                        </div>
+                        </label>
+                        <div class="col-md-10 col-sm-10 col-xs-12">
+                            <input id="cacheurl" name="cacheurl" type="text" class="form-control" ng-model="deliveryService.cacheurl" maxlength="1024">
+                            <small class="input-error" ng-show="hasPropertyError(cacheConfig.cacheurl, 'maxlength')">Too Long</small>
+                            <aside class="current-value" ng-if="settings.isRequest" ng-show="open() && deliveryService.cacheurl != dsCurrent.cacheurl">
+                                <h3>Current Value</h3>
+                                <pre>{{::dsCurrent.cacheurl}}</pre>
+                            </aside>
+                        </div>
                     </div>
-                </div>
-
-                <div class="form-group" ng-class="{'has-error': hasError(deliveryServiceForm.trRequestHeaders), 'has-feedback': hasError(deliveryServiceForm.trRequestHeaders)}">
-                    <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="trRequestHeaders">Traffic Router Log Request Headers<div class="helptooltip">
-                        <div class="helptext">
-                            List of header keys. One header key per line. Listed headers will be included in Traffic Router access log entries under the <samp>rh=</samp> token.
+                </ng-form>
+            </fieldset>
+            <fieldset>
+                <legend ng-class="{'fieldset-error': routingConfig.$invalid}" ng-click="showRoutingConfig = !showRoutingConfig">Routing Configuration Settings <i class="fa fa-caret-up" ng-show="!showRoutingConfig"></i><i class="fa fa-caret-down" ng-show="showRoutingConfig"></i></legend>
+                <ng-form name="routingConfig" ng-show="showRoutingConfig">
```

Comment:
A lot of these routing configuration changes don't get used by ANY_MAP Delivery Services - should they still be presented to the user? e.g. Geolocation Provider is used by TR to route client requests, but since TR doesn't route ANY_MAPs, the field has no effect on them.

Reply:
good point. these fields have always been present on the any_map form and like you said, probably ignored.

Reply:
can you create a separate issue for this as these fields have been present on the any map form for so long and i'd like to address their removal separate from this PR. good catch!

Reply:
It's already tracked by #1943

Reply:
oh wow. look at that. :)